assets/sass/patterns/organisms/hero-banner.scss

```diff
@@ -124,7 +124,7 @@
     text-decoration: none;
 
     &:hover {
-      color: $color-primary-dark;
+      @include color-link-hover();
```

Comment:
@chris-huggins Are we having underline on hover or color change for MSA's?

Reply:
I like the underline. I think we should also have a slightly darker blue on hover. Is that the `color-link-hover` you've suggested? I can't see a colour change on hover at the moment.